File: app/src/main/java/com/mj/richedittext/MainActivity.java
package com.mj.richedittext;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;

import jp.wasabeef.richeditor.RichEditor;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        final RichEditText findViewById = findViewById(R.id.preview);
        RichEditText mainEditor = findViewById(R.id.mainEditor);
        findViewById.setReadOnly(true);
        findViewById.hideMenuBar();
        mainEditor.setOnTextChangeListener(new RichEditor.OnTextChangeListener() {
            @Override
            public void onTextChange(String text) {

                findViewById.setText(text);
            }
        });
    }

}

File: app/src/main/java/com/mj/richedittext/MainActivity.kt
package com.mj.richedittext

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val findViewById = findViewById<RichEditText>(R.id.preview)
        findViewById.setReadOnly(true)
        findViewById.hideMenuBar();

        mainEditor.setOnTextChangeListener { text: String? ->

            findViewById.text = text;
        }


    }
}